src/test/java/com/epam/page/object/generator/integrationalTests/MainTest.java

```diff
@@ -172,4 +172,63 @@ public void pageObjectsGenerator_GenerateDropdownElementWithInnerElements() thro
         pog.generatePageObjects();
     }
 
+    @Test(expected = ValidationException.class)
+    public void pageObjectGenerator_notExistenceRootTitle() throws Exception {
```

Comment:
Does codestyle allow to use underscrores in method names? In general it prohibited for java source code, and if codestyle does not explicitly allow them, underscores should be avoided.

Reply:
Closing since Aleksandr already answered